core/meta/src/TClass.cxx

```diff
@@ -3451,38 +3451,31 @@ const char *TClass::GetSharedLibs()
 
 TList *TClass::GetListOfBases()
 {
-   if (!fBase) {
-      if (fCanLoadClassInfo) {
```

Comment:
Also the commit mentioned a reduction in scope of the lock, as far as I can tell it is actually the same what am I missing?